forge/drivers/localfs.js

```diff
@@ -4,6 +4,9 @@ const { join, isAbsolute, dirname, sep: pathSeparator, parse } = require('path')
 const canary = 'ROOT_DIR_CANARY'
 
 async function readDirSize (dir) {
+    if (!fs.existsSync(join(dir))) {
```

Comment:
Why is there a `join` in here?

Reply:
oh that looks off.  Lemmie check that over.

Reply:
Removed in [fd7ff47](https://github.com/flowforge/flowforge-file-server/pull/37/commits/fd7ff47976a204043584e19cd2e4e6138772b12a)